Modules/Microsoft365DSC/DSCResources/MSFT_IntuneDeviceEnrollmentPlatformRestriction/MSFT_IntuneDeviceEnrollmentPlatformRestriction.schema.mof

```diff
@@ -23,21 +23,21 @@ class MSFT_DeviceEnrollmentPlatformRestriction
 [ClassVersion("1.0.0.0"), FriendlyName("IntuneDeviceEnrollmentPlatformRestriction")]
 class MSFT_IntuneDeviceEnrollmentPlatformRestriction : OMI_BaseResource
 {
-    [Write, Description("Identity of the device enrollment platform restriction.")] String Identity;
-    [Key, Description("Display name of the device enrollment platform restriction.")] String DisplayName;
+    [Key, Description("Identity of the device enrollment platform restriction.")] String Identity;
+    [Write, Description("Display name of the device enrollment platform restriction.")] String DisplayName;
```

Comment:
I think this should be Required. We need a DisplayName if the ID does not exist.